<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创新实验平台考勤系统</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>231</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="logInButton">
      <property name="text">
       <string>登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signInButton">
      <property name="text">
       <string>签到</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="TrackButton">
      <property name="text">
       <string>训练特征值</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="hostActiveButton">
      <property name="text">
       <string>签到机认证</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="FaceLoginBtn">
      <property name="text">
       <string>管理员刷脸登录</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
